website_and_docs/content/documentation/getting_started/installing_selenium_libraries.zh-cn.md

```diff
@@ -3,19 +3,10 @@ title: "安装 Selenium 库"
 linkTitle: "安装 Selenium 库"
 weight: 1
 description: >
-  Setting up the Selenium library for your favourite programming language.
+为您喜爱的编程语言设置Selenium库.
```

Comment:
Same as above!

Reply:
Sorry for the mistake. let me configure latest hugo on my local to validate that